### PR TITLE
added faq for sawtooth-default-poet.yaml setup

### PR DESCRIPTION
**Pull request by raneja14**
The docker-compose file - sawtooth-default-poet.yaml in the
repository is not able to bring up a network always.
Few validators are left out(unregistered) at times. Subsequent
transaction submission leads to invalid blocks in validity
check at the PoET engine -

Signed-off-by: raneja14 <raneja7@gmail.com>

### CODE REVIEW

File: generator/source/faq/docker.rst
Comment (danintel) [inline generator/source/faq/docker.rst]: I suggest:
`Why doesn't sawtooth-default-poet.yaml start the network successfully on subsequent runs?`

File: generator/source/faq/docker.rst
Comment (danintel) [inline generator/source/faq/docker.rst]: Change "For example" to "For example the following removes containers and volumes"

Also break the line up into <= 80 characters.

**Review by danintel — CHANGES_REQUESTED** [review]

File: generator/source/faq/docker.rst
Comment (raneja14) [reply generator/source/faq/docker.rst]: I have made the necessary changes and updated the PR .

**Review by danintel — APPROVED** [review]

**Review by danintel — APPROVED** [review]

**Review by arsulegai — APPROVED** [review]

**Comment by arsulegai** [comment]
Can we get this PR merged, so that website's FAQ reflects it?

**Review by agunde406 — APPROVED** [review]

**Comment by danintel** [comment]
I just merged it. It needed a second approver, and 20 days later, we have two approvals so I can merge. Thanks!

**Pull request merged**
